src/components/topic/TopicDirective.js

```diff
@@ -31,7 +47,12 @@ goog.require('ga_topic_service');
               }
             });
 
-            gaTopic.loadConfig().then(function() {
+            $rootScope.$on('$translateChangeEnd', function() {
+              scope.topics = gaTopic.getTopics();
```

Comment:
I think it makes more sense to use to `translateTopics` here and line 56, to avoid this translation loop is executed on every digest cycle.